projects/reservoir/fv3/tests/test_save_ranks_cli.py

```diff
@@ -0,0 +1,41 @@
+import pytest
```

Comment:
I think you could delete this test file, unless you want to make sure the CLI stays backwards-compatible and the argument names don't change. It's not necessary to test `argparse`. What I was talking about earlier was the argument handling logic that came after the argparse calls, outside of the new unit-tested routines.

Reply:
Ok I see. I thought you were referring to the possibility of the multiple variable names getting messed up and ending up saving data with just one, or some error like that.